apps/service-catalog-e2e/src/e2e/app.cy.ts

```diff
@@ -0,0 +1,13 @@
+// import { getGreeting } from '../support/app.po';
```

Comment:
skal denne filen være med siden alt er kommentert?
Kanskje cypress e2e mappen ikke skal være med i det hele tatt siden vi går for Playwright? Kan evt dobbeltsjekke med Jeff

Reply:
Ja det var det jeg tenkte, så slettet den nå. Var også den som hindret deploy i å fungere 🥴